src/SB/Core/x/xCollide.cpp

```diff
@@ -251,7 +264,398 @@ void xParabolaRecenter(xParabola* p, F32 newZeroT)
+{
+    xVec3 vec;
+    xMat3x3Transpose(a, b);
+    xMat3x3RMulVec(&vec, a, &b->pos);
+    xVec3Inv(&a->pos, &vec);
+}
+
+F32 xMat3x3LookVec3(xMat3x3& mat, const xVec3& at)
+{
+    F32 len = at.length();
+    if (len >= -0.0000099999997f && len <= 0.0000099999997f)
+    {
+        mat = g_I3;
+        len = 0.0f;
+    }
+    else
+    {
+        mat.at = at;
+        mat.at *= 1.0f / len;
+        F32 absx = xabs(mat.at.x);
+        F32 absy = xabs(mat.at.y);
+        F32 absz = xabs(mat.at.z);
+        if (absx < absy && absx < absz)
+        {
+            mat.right.assign(0.0f, absz, -absy);
+        }
+        else
+        {
+            if (absy < absz)
+            {
+                mat.right.assign(-mat.at.z, 0.0f, mat.at.x);
+            }
+            else
+            {
+                mat.right.assign(mat.at.y, -mat.at.x, 0.0f);
+            }
+        }
+        mat.right.normalize();
+        mat.up = mat.at.cross(mat.right);
+    }
+    return len;
+}
+
+void xModelAnimCollRestore(const xModelInstance&)
+{
+}
+
+void xModelAnimCollApply(const xModelInstance&)
+{
+}
+
+bool xModelAnimCollDirty(const xModelInstance&)
+{
+    return false;
+}
+
+// Make these into inline definitions somewhere appropriate later
+xVec3 xVec3::operator+(const xVec3& v) const
+{
+    xVec3 vec = *this;
+    vec += v;
+    return vec;
+}
+
+xVec3& xVec3::operator+=(const xVec3& v)
+{
+    x += v.x;
+    y += v.y;
+    z += v.z;
+    return *this;
+}
+
+xVec3 xVec3::get_abs() const
+{
+    xVec3 vec = *this;
+    return vec.set_abs();
+}
+
+xVec3& xVec3::set_abs()
+{
+    x = xabs(x);
+    y = xabs(y);
+    z = xabs(z);
+    return *this;
+}
+
+F32 xVec3::dot(const xVec3& v) const
+{
+    return x * v.x + y * v.y + z * v.z;
+}
+
+xVec3 xVec3::cross(const xVec3& v) const
+{
+    return xVec3();
+}
+
+xVec3& xVec3::normalize()
+{
+    *this /= length();
+    return *this;
+}
+
+xVec3 xVec3::operator/(F32 f) const
+{
+    xVec3 vec = *this;
+    vec /= f;
+    return vec;
+}
+
+void xParabolaEvalVel(const xParabola*, xVec3*, F32)
+{
+}
+
+void xParabolaEvalPos(const xParabola*, xVec3*, F32)
+{
+}
+
+bool xSphereHitsOBB(const xSphere&, const xBox&, const xMat4x3&)
+{
+    return false;
+}
+
+bool xSphereHitsSphere(const xVec3&, F32, const xVec3&, F32)
+{
+    return false;
+}
+
+void xQuickCullForRay(xQCData*, const xRay3*)
+{
+}
+
+void xQuickCullForBox(xQCData*, const xBox*)
+{
+}
+
+U8 xSphereHitsCapsule(const xVec3&, F32, const xVec3&, const xVec3&, F32)
+{
+    return 0;
+}
+
+F32 xVec2::length2() const
+{
+    return x * x + y * y;
+}
+
+xVec2 xVec2::operator-(const xVec2& v) const
+{
+    xVec2 vec = *this;
+    vec -= v;
+    return vec;
+}
+
+xVec2& xVec2::operator-=(const xVec2& v)
+{
+    x -= v.x;
+    y -= v.y;
+    return *this;
+}
+
+// Get these assignment operators to exist
+static void tempReferenceWeakDefinitions(RwLine* a, nodeInfo* b, RwBBox* c, xCollis* d)
+{
+    *a = *a;
+    *b = *b;
+    *c = *c;
+    *d = *d;
+}
```

Comment:
This particular change I'm not sure about. Personally, I'm not a fan of these short-term matches just for getting the matchingness percentage up since it creates cleanup work for later on. I'm open to discussing it though if someone feels differently.

Reply:
@stravant For the sake of getting this in, please remove this function in line with the recently merged guidelines (#554) on avoiding placeholder functions like this for short-term matchingness increases.

Reply:
Yeah, I don't need this anymore, I was just trying out whether that worked as an approach to make it link.